ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2616,3 +2616,59 @@ periodics:
       resources:
         requests:
           cpu: 500m
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: '@daily'
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-trigger-managed-clonerefs-mabc
+  spec:
+    containers:
+    - command:
+      - /bin/bash
+      args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        export KUBECONFIG='/etc/build-farm-credentials/kubeconfig'
+
+        curl -sL 'https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux-amd64' >/tmp/jq
+        chmod +x /tmp/jq
+
+        managed_clonerefs=$(oc -n ci get mabc/managed-clonerefs -o json)
+        if [ -z "$(/tmp/jq -r '.status.state' <<<$managed_clonerefs)" ]; then
+            echo 'mabc/managed-clonerefs is running already'
+            exit 0
+        fi
+
+        oc -n ci delete --cascade=foreground --wait=true mabc/managed-clonerefs
+        echo "mabc/managed-clonerefs deleted"
+
+        oc -n ci apply --wait=true -f 'clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml'
```

Comment:
This will failed. You need to take care of the `serviceAccount` as well.

Reply:
I am not sure. Let's config that the `config-update` has rbacs to manage the mabc resources.

Reply:
This is the output from a pod running `ocp/cli-jq:latest` on `multi01`:
```bash
sh-4.4# oc whoami
system:serviceaccount:ci:config-updater
sh-4.4# oc auth can-i '*' mabc
yes
```

Reply:
I don't agree on using the config-updater account for that... I would prefer having a different service account just for managing the mabc for that specific case. @hongkailiu Can add more context here.

Reply:
In the end the only thing this periodic does is applying and deleting a resource. Isn't the `config-updater` SA meant to handle such a case? `applyconfig` does more or less the same and it leverages `configu-updater` capabilities.

Reply:
This job is not related to the applyconfig. Instead this job is doing a specific task on specific resources. We should use more permissions that the job needs.